Templates/Python/Model/ComplexType.py.tt

```diff
@@ -86,11 +86,11 @@ class <#=complex.Name.ToUpperFirstChar()#>(OneDriveObjectBase):
         Gets and sets the <#=property.Name#>
         
         Returns: 
-            :class:`<#=property.Type.GetTypeString()#><onedrivesdk.model.<#=property.Type.GetTypeString().ToUnderscore()#>.<#=property.Type.GetTypeString()#>>`:
+            :class:`<#=property.Type.GetTypeString()#><microsoft.graph.model.<#=property.Type.GetTypeString().ToUnderscore()#>.<#=property.Type.GetTypeString()#>>`:
```

Comment:
There shouldn't be any hard coded namespaces. This should have come out of the Vipr object models.